src/popup/components/mnemonicPhrase/ConfirmMnemonicPhrase.tsx

```diff
@@ -1,12 +1,13 @@
-import React, { useRef, useState } from "react";
```

Comment:
convert this view to use Formik. Needed a bit of refactoring, unfortunately doesn't save us a whole lot of work bc we still have to do most of it manually